docs/Limitations.md

```diff
@@ -216,6 +219,27 @@ Equivalent functionality can be achieved with the newer docker networking comman
 See more documentation at
 [docs.docker.com](https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/).
 
+## Storage limitations
+
+### Block devices
+
+Currently, block devices are mounted on the host and passed to the guest VM
+using a shared filesystem, `virtiofs`. This has consequences notably in terms of
+performance, although it should generally behave as expected as far as
+functionality is concerned.
+
+A [proposal to address this](https://github.com/egernst/kata-containers/blob/da-proposal/docs/design/direct-assign-volume.md)
+is under consideration.
+
+### Kubernetes `volumeMounts.subPaths`
+
+Kubernetes `volumeMount.subPath` is not supported by Kata Containers at the
+moment.
```

Comment:
Since subpaths are not part of runtime spec, I would just remove 'at the moment'

```suggestion
Kubernetes `volumeMount.subPath` is not supported by Kata Containers.
```